utils/common/src/main/java/org/apache/brooklyn/util/javalang/Reflections.java

```diff
@@ -204,6 +209,7 @@ public Object loadInstance(String classname) throws ReflectionNotFoundException,
     }
 
     /** does not look through ancestors of outer class */
+    /** @deprecated since 0.11.0, use {@link org.apache.brooklyn.util.core.ClassLoaderUtils} in a combination with {@link #invokeConstructorFromArgs(Class, Object...)} instead */
```

Comment:
Alternatively we could allow `Reflections to be constructed with a `ClassLoaderUtils`, and deprecate the method that creates it with a `ClassLoader`. We'd also deprecate the static methods that take a `ClassLoader`?